assets/js/sitemap-definitions.js

```diff
@@ -49,11 +49,12 @@
             return result
         }
 
-        data.options.data['itemData'] = iterator($items)
+        var fieldName = $('div[data-control=sitemap-item-editor]').data('fieldName')
```

Comment:
Just FYI, every single supported browser supports the use of `let` and `const`, so for all new JS code, we should use `const` unless we're very sure the variable needs to change, just to ensure there's no unintended side effects with the use of a variable.

```suggestion
        const fieldName = $('div[data-control=sitemap-item-editor]').data('fieldName')
```